flink-benchmark/run_and_verify.sh

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2022 The Feathub Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+cd "$(dirname "$0")"
+PROJECT_DIR=$(cd "$(pwd)/.."; pwd)
+source "${PROJECT_DIR}"/tools/utils.sh
+
+docker-compose up -d
+wait_for_port 8081 "Flink Cluster"
+
+for BENCHMARK_SCRIPT in ./*.py; do
+  echo "Running benchmark ${BENCHMARK_SCRIPT}..."
+  python "${BENCHMARK_SCRIPT}"
```

Comment:
We probably want to run the benchmark script in the nightly build so that we can catch and fix broken scripts early. But we also don't want these benchmark scripts to take too much time in the nightly build.

How about we update script and README properly so that users can easily run benchmark to get meaningful results, whereas the nightly build will run the benchmark script for only a small number of inputs?